src/scripts/globalize.js

```diff
@@ -38,6 +40,43 @@ import { currentSettings as userSettings } from './settings/userSettings';
         return fallbackCulture;
     }
 
+    export function getIsRTL() {
+        return isRTL;
+    }
+
+    function checkAndProcessDir(culture) {
+        isRTL = false;
+        for (const lang of RTL_LANGS) {
```

Comment:
I think for `RTL_LANGS` we should use the full culture string to simplify  this check and avoid this looping. Then the check could just be `RTL_LANGS.includes(culture)`.

Reply:
The problem is that the culture parameter is sometimes of the format language, as in en or ar or pe, and sometimes is of the format en_US, ar_SA etc. Also checking for language is easier than checking for language + locale combinations. For arabic, there are 28 different locales.